pages/book/[id].js

```diff
@@ -0,0 +1,33 @@
+import { useRouter } from "next/router";
+import { books } from "../../lib/books.js";
```

Comment:
Instead of importing books again you should probably pass the data as a prop. You could also save the data in a state.